### ISSUE TITLE
Upgrade BEMCheckBox Native Framework from 1.4.1 to 2.0.0

### ISSUE THREAD

**Issue by mrbiggred** [saturdaymp/XPlugins.iOS.BEMCheckBox#12]
There hasn't been a new release of [BEMCheckBox](https://github.com/Boris-Em/BEMCheckBox) and it appears that no new release is [forth coming](https://github.com/Boris-Em/BEMCheckBox/issues/89).  There are a couple fixes that have been [committed](https://github.com/Boris-Em/BEMCheckBox/commits/master) to since the official BEMCheckBox [1.4.1 release](https://github.com/Boris-Em/BEMCheckBox/releases/tag/1.4.1).  It would be nice to incorporate those changes.

**Edit (Aug 8, 23):** The BEMCheckBox has been forked and a new a [v2.0.0](https://github.com/saturdaymp/BEMCheckBox/releases/tag/v2.0.0) release created.  Try to use this release.

**Comment by racoltacalin** [comment]
Hi, is there any news about this package?

**Comment by mrbiggred** [comment]
Thanks for reaching out @racoltacalin.  I just recently finished up the automated [build](https://github.com/saturdaymp/XPlugins.iOS.BEMCheckBox/actions) for the Xmarain part of the checkbox.  Now I need to get a new build of BEMCheckBox and create a native reference.  I was trying to automate this process using GitHub Actions but if there is a demand for a new release I will might just [manually](https://nftb.saturdaymp.com/today-i-learned-how-to-create-a-xamarin-ios-binding-for-objective-c-libraries-part-1-compiling-the-objective-c-library/) create the native extension.

I used to use TeamCity to [build](https://nftb.saturdaymp.com/today-i-learned-how-to-automate-objective-c-builds-in-teamcity/) the BEMCheckBox native extension but no longer have access to this setup anymore and TeamCity Cloud does not support Mac [runners](https://www.jetbrains.com/help/teamcity/cloud/supported-platforms-and-environments.html).  GitHub Actions has Mac runners but I'm not very familiar with compiling Objective-C/Swift applications.  If you have any advice that would be much appreciated.
